### ISSUE TITLE
Remove old versions from readme.txt change log

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#613]
See the [plugin guidelines](https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#how-many-versions-should-i-keep-in-my-changelog):

>Always keep the current major release in your change log. For example, if your current version is 3.9.1, you’ll want that and 3.9 in the change log. Older versions should be removed and migrated to a changelog.txt file. That will allow them to be accessible to users, while keeping your readme shorter and more pertinent. At most, keep the most recent version of your plugin and one major version back in your readme’s changelog. Your changelog.txt will not be visible within the WordPress.org Plugin Directory, but that’s okay. Most users just want to know what’s new.

**Comment by JDGrimes** [comment]
Same for the update notices, too, I guess.

**Comment by JDGrimes** [comment]
For the update notices, I was intended to just remove them. I was thinking at first that we only need to keep the ones for the active versions:

![wordpoints_ _wordpress_plugins_-_2017-02-18_08 48 32](https://cloud.githubusercontent.com/assets/4005415/23093642/10df741a-f5b7-11e6-8a2d-335d498efa12.png)

Or rather, that the only ones that users would ever see would be the ones for those newer than the earliest active version, which is 2.0. So I was thinking we needed to keep 2.0.1 and later.

However, last night I happened to think that probably only the latest version's update notice will ever be displayed to users, because that is the only version that they can update to.

But then, when a user doesn't have a sufficient version of WordPress to run a plugin, WordPress doesn't show them the update at all. But maybe they show them the update for the most recent version of the plugin that they can run?

So now I'm not sure. I guess that either way, it probably makes sense to keep them from 2.0.1 and later, in case plugins are utilizing them or providing such a granular update feature, even if WordPress core doesn't.
